src/main/resources/tags/info/bungee.tag

```diff
@@ -3,6 +3,10 @@ aliases: bungeecord, velocity
 
 ---
 
-The Geyser and Floodgate BungeeCord plugins only need to be installed on the BungeeCord proxy unless you intend to use the Floodgate API. That way, you need Floodgate on the "backend" servers too.
-Installing Floodgate on the backend servers also makes it so that Bedrock players don't have to switch backend servers for their skins to start displaying.
-Run `!!api` in <#613194762249437245> for more information regarding the Floodgate API.
+For the most basic installation, the Geyser and Floodgate Bungeecord or Velocity plugins only need to be installed on the proxy.
+
+For more andvance/complete features, you may also install Floodgate on the backend servers:
+***-*** It makes it so that Bedrock players don't have to switch backend servers for their skins to start displaying (once they're uploaded to the skin server).
+***-*** If you intend to use certain plugins such as [BedrockPlayerManager](https://www.spigotmc.org/resources/bedrock-player-managment-for-floodgate-geyser.82278/)
+***-*** If you wish to use the API for your own puropses. Run `!!api` in <#613194762249437245> for more information regarding the Floodgate API.
```

Comment:
```suggestion
***-*** If you wish to use the API for your own purposes, run `!!api` in <#613194762249437245> for more information regarding the Floodgate API.
```

Reply:
Separate sentences are better imo.

Reply:
Also your edit makes it no longer a reason to complete the thought at the start of the list with an additional note, but makes that entry it's own thing.

Reply:
the way I have it would read as
 `For more advanced/complete features, you may also install Floodgate on the backend servers if you wish to use the API for your own purposes. Run `!!api` in <#613194762249437245> for more information regarding the Floodgate API.`

the way you have it would read as:
`
For more advanced/complete features, you may also install Floodgate on the backend servers.

If you wish to use the API for your own purposes. Run `!!api` in <#613194762249437245> for more information regarding the Floodgate API.`

See the disconnect? (I do need to correct the spelling of "purposes" regardless.)

Reply:
Oh, I see, I thought this was a mistake. Maybe make it clearer that it isn’t?

Reply:
Maybe do `for your own purposes - run`

Reply:
I've altered the way the line reads a little bit.